src/main/java/Duke.java

```diff
@@ -1,10 +1,49 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static void printList(Task[] list) {
+        System.out.println("     Here are the tasks in your list:");
+        for(int i = 0; i < list.length; i ++) {
+            if(list[i] == null) {
+                break;
+            }
+            System.out.println("     " + (i + 1) + "[" + list[i].getStatusIcon() + "] " + list[i].description);
+        }
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Task[] list = new Task[100];
```

Comment:
You may want to consider setting the number 100 as a constant instead. This can help to give more meaning to the number.
Reference: [ https://nus-cs2113-ay1920s2.github.io/website/book/codeQuality/maximiseReadability/basic/avoidMagicNumbers/](https://nus-cs2113-ay1920s2.github.io/website/book/codeQuality/maximiseReadability/basic/avoidMagicNumbers/)